src/app/AttributeValueEncoder.h

```diff
@@ -169,17 +140,16 @@ class AttributeValueEncoder
         if (err == CHIP_NO_ERROR)
         {
             // The Encode procedure finished without any error, clear the state.
-            mEncodeState = AttributeEncodeState();
+            mEncodeState.Reset();
         }
         return err;
     }
 
     bool TriedEncode() const { return mTriedEncode; }
 
-    /**
-     * The accessing fabric index for this read or subscribe interaction.
```

Comment:
Again, why removing the API docs?  The key part is not documenting that this is the accessing fabric index (which is obvious from the name, sure), but _what it's the accessing fabric index for_, in spec terms.